Sources/NIOCore/ByteBuffer-multi-int.swift

```diff
@@ -0,0 +1,1053 @@
+        var v8: T8
+        var v9: T9
+        var v10: T10
+        switch endianness {
+        case .big:
+            v1 = value1.bigEndian
+            v2 = value2.bigEndian
+            v3 = value3.bigEndian
+            v4 = value4.bigEndian
+            v5 = value5.bigEndian
+            v6 = value6.bigEndian
+            v7 = value7.bigEndian
+            v8 = value8.bigEndian
+            v9 = value9.bigEndian
+            v10 = value10.bigEndian
+        case .little:
+            v1 = value1.littleEndian
+            v2 = value2.littleEndian
+            v3 = value3.littleEndian
+            v4 = value4.littleEndian
+            v5 = value5.littleEndian
+            v6 = value6.littleEndian
+            v7 = value7.littleEndian
+            v8 = value8.littleEndian
+            v9 = value9.littleEndian
+            v10 = value10.littleEndian
+        }
+
+        let spaceNeeded = Int(MemoryLayout<T1>.size) &+ Int(MemoryLayout<T2>.size) &+ Int(MemoryLayout<T3>.size) &+ Int(MemoryLayout<T4>.size) &+ Int(MemoryLayout<T5>.size) &+ Int(MemoryLayout<T6>.size) &+ Int(MemoryLayout<T7>.size) &+ Int(MemoryLayout<T8>.size) &+ Int(MemoryLayout<T9>.size) &+ Int(MemoryLayout<T10>.size)
+        return self.writeWithUnsafeMutableBytes(minimumWritableBytes: spaceNeeded) { ptr -> Int in
+            var offset = 0
+            let basePtr = ptr.baseAddress! // safe: pointer is non zero length
+            (basePtr + offset).copyMemory(from: &v1, byteCount: MemoryLayout<T1>.size)
+            offset = offset &+ MemoryLayout<T1>.size
+            (basePtr + offset).copyMemory(from: &v2, byteCount: MemoryLayout<T2>.size)
+            offset = offset &+ MemoryLayout<T2>.size
+            (basePtr + offset).copyMemory(from: &v3, byteCount: MemoryLayout<T3>.size)
+            offset = offset &+ MemoryLayout<T3>.size
+            (basePtr + offset).copyMemory(from: &v4, byteCount: MemoryLayout<T4>.size)
+            offset = offset &+ MemoryLayout<T4>.size
+            (basePtr + offset).copyMemory(from: &v5, byteCount: MemoryLayout<T5>.size)
+            offset = offset &+ MemoryLayout<T5>.size
+            (basePtr + offset).copyMemory(from: &v6, byteCount: MemoryLayout<T6>.size)
+            offset = offset &+ MemoryLayout<T6>.size
+            (basePtr + offset).copyMemory(from: &v7, byteCount: MemoryLayout<T7>.size)
+            offset = offset &+ MemoryLayout<T7>.size
+            (basePtr + offset).copyMemory(from: &v8, byteCount: MemoryLayout<T8>.size)
+            offset = offset &+ MemoryLayout<T8>.size
+            (basePtr + offset).copyMemory(from: &v9, byteCount: MemoryLayout<T9>.size)
+            offset = offset &+ MemoryLayout<T9>.size
+            (basePtr + offset).copyMemory(from: &v10, byteCount: MemoryLayout<T10>.size)
+            offset = offset &+ MemoryLayout<T10>.size
+            assert(offset <= ptr.count)
+            return offset
+        }
+    }
+
+}
+
+
+/* --- AUTO-GENERATION SCRIPT FOR THIS FILE ---
+#!/bin/bash
```

Comment:
As much as a I love script which generates itself, can move this to its own file?

Reply:
I can do that too, where would you like it to live?

Reply:
`scripts` should be fine

Reply:
@glbrntt done, it's its own script now.